projects/step-frontend/src/lib/modules/timeseries/dashboard/time-series-dashboard.component.html

```diff
@@ -1,20 +1,41 @@
 <ng-container *ngIf="performanceViewSettings">
-  <step-ts-filter-bar
-    *ngIf="context"
-    [context]="context"
-    class="mat-card"
-    [performanceViewSettings]="performanceViewSettings"
-    [defaultFilterOptions]="settings.filterOptions"
-    [initialFilters]="contextualFilterItems"
-    (onFiltersChange)="handleFiltersChange($event)"
-    (onFilterSettingsChange)="handleFilteringChange($event)"
-    (onGroupingChange)="handleGroupingChange($event)"
-  >
-  </step-ts-filter-bar>
+  <div>
+    <div class="filter-bars-container" [style.gap]="compareModeEnabled ? '12px' : '0'">
+      <div class="bar mat-card">
+        <step-ts-filter-bar
+          *ngIf="context"
+          [context]="context"
+          [defaultFilterOptions]="settings.filterOptions"
+          [initialFilters]="contextualFilterItems"
+          (onTimeRangeChange)="handleTimeRangeChange($event)"
+          [timeRangeOptions]="settings.timeRangeOptions"
+          [activeTimeRange]="settings.activeTimeRange"
+          [compactView]="compareModeEnabled"
+        >
+        </step-ts-filter-bar>
+        <div *ngIf="compareModeEnabled" class="stamp">BASE</div>
+      </div>
+      <div class="bar mat-card" [style.flexGrow]="compareModeEnabled ? 1 : 0">
```

Comment:
same as mentioned above with the added note, that I would add compare-mode-enabled somewhere higher level so one property could be used by all components.

Reply:
@neogucky if you mean pushing this compareModeEnabled through all components via @Input, I don't think it's a good ideea. We would have to put ngOnChanges everywhere, which I don't like. I'd rather prefer to react on it via the state service, which later can be replaced with anything, state, service, redux, etc.

Reply:
No, I mean not to set css inside the template but instead to place a class. In this case class should be placed in the top level div since both occurences would be able to be adressed by 
`
.compare-mode-enabled {
    .mat-card { ... }
    .filter-bars-container { ... }
}
`